pandas/tests/indexes/multi/test_equivalence.py

```diff
@@ -254,12 +251,10 @@ def test_is_():
     mi4 = mi3.view()
 
     # GH 17464 - Remove duplicate MultiIndex levels
-    with tm.assert_produces_warning(FutureWarning):
-        mi4.set_levels([list(range(10)), list(range(10))], inplace=True)
+    mi4 = mi4.set_levels([list(range(10)), list(range(10))])
```

Comment:
Similar comment - might be able to just remove

Reply:
This is a bit different, would like to keep it